README.md

```diff
@@ -1,71 +1,52 @@
-# Getting Started with Create React App
+### BookStore ###
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+My Bookstore app is a website to show a list of books using a given design and using an API, React, Redux, Javascript and Css to build and style it.
 
-## Available Scripts
+## Built With
 
-In the project directory, you can run:
+# HTML, CSS, JavaScript
+# React, Redux
 
-### `npm start`
+## Live Demo
+not yet...
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Getting started
+To get a local copy up and running follow these simple example steps. (NPM)
```

Comment:
- [ ] Good job on the improvements made but as requested by the previous reviewer, please use this UPDATED [README TEMPLATE](https://github.com/microverseinc/readme-template) as provided by Microverse in order to make your README look more professional 😃 👍